### ISSUE TITLE
IPv6 support

### ISSUE THREAD

**Issue by bi4o4ek** [JoeDog/siege#72]
If host has only AAAA record (ipv6 address) then siege fails with errors
```
[error] descriptor table full sock.c:108: Too many open files
[error] descriptor table full sock.c:108: Too many open files
```

Command `siege -b https://2a02:6b8:0:1a79::f90 -v` also raises these errors.

**Comment by JoeDog** [comment]
Yeah, IPv6 isn't supported. (I've been hoping someone would send me a patch.

I'll add it soon but it's not a high priority.

Jeff

**Comment by bi4o4ek** [comment]
ping?)

**Comment by florinpapa** [comment]
Added this pull request for IPv6 support.

https://github.com/JoeDog/siege/pull/100
